src/main/java/frc/robot/commands/Balancing/PIDSwitchBalancer.java

```diff
@@ -0,0 +1,109 @@
+package frc.robot.commands.Balancing;
+
+import com.fasterxml.jackson.databind.JsonSerializable.Base;
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.subsystems.drivetrain.BaseDrivetrain;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+
+public class PIDSwitchBalancer extends CommandBase {
```

Comment:
Small style thing but should probably mention `Command` in the name; `SwitchingBalancerCommand`?